src/Aurora.Infrastructure/Services/DriverInitializer.cs

```diff
@@ -20,10 +20,10 @@ public DriverInitializer(IOptions<SeleniumConfig> options)
 
         public Task<RemoteWebDriver> Initialize()
         {
-            string seleniumLocation = _options.SeleniumLocation;
-            Uri uri = new Uri(seleniumLocation);
+            var seleniumLocation = _options.SeleniumLocation;
+            var uri = new Uri(seleniumLocation);
             var options = new FirefoxOptions();
-            RemoteWebDriver driver = new RemoteWebDriver(uri, options.ToCapabilities());
```

Comment:
Why wouldn`t you just change options to FirefoxOptions?

Reply:
you have added this part

Reply:
You have changed 3 variables to var, instead of changing one variable to FirefoxOptions

Reply:
I am a genius, I know